daemon/checkpoint.go

```diff
@@ -82,6 +83,23 @@ func (daemon *Daemon) CheckpointCreate(name string, config types.CheckpointCreat
 		return fmt.Errorf("Cannot checkpoint container %s: %s", name, err)
 	}
 
+	checkpointInfo := types.Checkpoint{
```

Comment:
Please define new struct, because `api/types` is for REST API

Reply:
For now, "docker checkpoint ls" just return "Checkpoint name", I think we need more useful information.
So, my original idea is that after this patch, "docker checkpoint ls <container name>" should print more information.
CheckpintName, Create Time, Exit
So I reuse "types.Checkpoint".

Do you still think that we need to define a new struct?
If yes, I'll redefine it.

Reply:
Ok, thank you for your review.
For one PR do one thing, I defined a new struct.